release-notes/1.0/1.0.8.md

```diff
@@ -21,6 +21,26 @@ Deployment of the November 2017 Update on Azure AppServices is in process. Becau
 
 ### Security Advisories
 
+Microsoft is releasing security advisories for .NET Core and ASP.NET Core. Details can be found in corresponding announcements in the [.NET Core](https://github.com/dotnet/announcements) and [ASP.NET Core](https://github.com/aspnet/announcements) repos.
+
+#### CVE-2017-8585 Malformed Certificate can cause Denial of Service
```

Comment:
This doesn't match the earlier announcement:

- [CVE-2017-8585 : Malformed Culture can cause application crash](https://github.com/dotnet/announcements/issues/34)
- [CVE-2017-11770 : Malformed Certificate can cause Denial of Service](https://github.com/dotnet/announcements/issues/44)